buildscript/build.go

```diff
@@ -27,6 +30,35 @@ var pkgNames = []string{
 	"release",
 }
 
+// minimumGoVersion must be prefixed with v to be parsed by golang.org/x/mod/semver
+var minimumGoVersion = "v1.19.12"
+
+func CheckMinimumGoVersion(ctx *task.Context) error {
+	goVersionStr, err := runCmd(ctx, "go", "version")
+	if err != nil {
+		return fmt.Errorf("failed to get current go version: %w", err)
+	}
+
+	_, _ = ctx.Write([]byte(fmt.Sprintf("Found Go version \"%s\"\n", goVersionStr)))
```

Comment:
This is just to log to stdout?

Reply:
Yep, seems to be the way to do it in goke so it's formatted correctly.